xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/icons/default.iconset

```diff
@@ -1158,6 +1165,14 @@ xhtml_valid = xhtml_valid
 zoom = zoom
 zoom_in = zoom_in
 zoom_out = zoom_out
-filter = find
+
+## Deprecated since 16.2.0 to conform to the new naming convention.
+bullet_black = bullet_black
+credit-card = creditcards
+emoticon_smile = emoticon_smile
 fast-backward = control_rewind
-arrow_in = arrow_in
+fast-forward = control_fastforward
+floppy-disk = disk
+log-in=user_go
+log-out=user_go
+shopping-cart = cart
```

Comment:
Preferably those would be moved in a legacy module. But, I'm not sure we have an easy way to do that.

Reply:
There's the same kind of deprecated block in all icon themes. I'd rather have the same structure as the one we already have in translation files. It keeps things easy to maintain and understand, even if it might not the most optimized.